### PR TITLE
Docker pushbox ng

### PR DESCRIPTION
**Pull request by rfk**
Fixes https://github.com/mozilla/fxa-dev/issues/381

WIP while I do a test deploy and shake out the kinks.

### CODE REVIEW

**Comment by rfk** [comment]
@eoger this is now deployed and seems to be running correctly on https://pushbox3.dev.lcip.org, could you please try out the pushbox-related features and let me know if it seems to be working correctly?

**Comment by eoger** [comment]
Works for me! 👍

**Comment by rfk** [comment]
Thanks @eoger!  I've updated the branch to remove the pushbox envs since it will now be enabled by default.  @mozilla/fxa-devs r?

**Review by vladikoff — DISMISSED** [review]
Ryan, there seems to be a problem with `OpenSSL`. 

If I check the logs with `docker logs pushbox`, I get a lot of

`Jul 09 13:53:48.096 WARN Error: HttpDispatch(HttpDispatchError { message: "The OpenSSL library reported an error" })
Jul 09 13:58:48.250 WARN Error: HttpDispatch(HttpDispatchError { message: "The OpenSSL library reported an error" })`

This can probably be fixed by including `ca-certificates` in the Docker image: https://github.com/mozilla/fxa-email-service/commit/0e172a9c4877409f9577bb69d0cca1be45775ddb#diff-3254677a7917c6c01f55212f86c57fbfR29

Ref: https://stackoverflow.com/questions/44728421/docker-and-the-openssl-library-reported-an-error-when-deployed

The other issue I noticed is that output is not `mozlog` per one weird successful log:

```
GET /v1/store/1b8ab34c05a54de591f9db642545f814/8db1639ed3be2575f58abd9e8595e56e?limit=100:
    => Matched: GET /v1/store/<user_id>/<device_id>?<options>
    => Outcome: Success
    => Response succeeded.
```

Do we need to fix that?

**Comment by rfk** [comment]
> Jul 09 13:53:48.096 WARN Error: HttpDispatch(HttpDispatchError { message: "The OpenSSL library
> reported an error" }) Jul 09 13:58:48.250 WARN Error: HttpDispatch(HttpDispatchError { message:
> "The OpenSSL library reported an error" })

These errors don't appear to be fatal, although I agree we should fix them.  I'm just pulling the public docker image for pushbox, so I guess they'll need to add the certificates thing as part of their build.  I'l file an issue and/or PR.

> The other issue I noticed is that output is not mozlog per one weird successful log:

FWIW I don't think we need to make this our problem, unless it's going to break something in the fxa-dev logging setup.  The pushbox folks will work through what they need to work through in order to get logging hooked up in production.

**Comment by rfk** [comment]
Upstream PR: https://github.com/mozilla-services/pushbox/pull/29

**Comment by rfk** [comment]
Thanks @vladikoff, I think I've fixed the errors by fixing the upstream docker image, and including explicit configuration for the SQS queue that pushbox wants to poll.  Could you please take another look?

If this looks OK, I will pop the topmost commit before merging, and remove the "pushbox*" dev boxes once it's confirmed working on latest.

File: aws/cloudformation/moz-single.json [hunk 1]
```diff
@@ -245,6 +245,7 @@
             " -e basket_queue_url=",         { "Ref": "FxaBasketAccountChangeQueue" },
             " -e oauth_queue_url=",          { "Ref": "FxaOAuthAccountChangeQueue" },
             " -e profile_queue_url=",        { "Ref": "FxaProfileAccountChangeQueue" },
+            " -e pushbox_queue_url=",        { "Ref": "FxaPushboxAccountChangeQueue" },
```
Comment (vladikoff) [inline aws/cloudformation/moz-single.json:248]: we might be missing this addition in https://github.com/mozilla/fxa-dev/blob/docker/aws/roles/cron_update/tasks/main.yml#L50

Comment (jrgm) [reply aws/cloudformation/moz-single.json:248]: Yeah, you have to put `pushbox_queue_url={{ pushbox_queue_url }}` in that mess of a line.

Comment (rfk) [reply aws/cloudformation/moz-single.json:248]: Aha, thanks, now I know to check ` /var/log/ansible/update.log` for errors too; added

**Review by vladikoff — APPROVED** [review]

**Pull request merged**